Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "auto_impl"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
This should be `0.2.0` as this pr is a functional change.

http://semver.org/#spec-item-6
> 6. Patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.

Reply:
I think it's pretty common in the Rust ecosystem to use the patch version effectively as the minor is post `1.0` when the crate is pre `1.0`, since we're only utilising two versions and aren't technically bound by semver anyways in `0.x`.

If there's code that was accepted in `0.1.1` that isn't in `0.1.2` then I've messed up.

Reply:
I see.